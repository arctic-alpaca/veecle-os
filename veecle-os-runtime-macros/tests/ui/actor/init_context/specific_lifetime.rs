#[veecle_os_runtime_macros::actor]
async fn macro_test_actor(
    #[init_context] _init_context: &'a u8,
) -> std::convert::Infallible {
    unreachable!("We only care about the code compiling.");
}

fn main() {}
