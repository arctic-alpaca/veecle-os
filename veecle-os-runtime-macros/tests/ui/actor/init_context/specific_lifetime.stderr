error: lifetimes on actor arguments must be anonymous or static
 --> tests/ui/actor/init_context/specific_lifetime.rs:3:37
  |
3 |     #[init_context] _init_context: &'a u8,
  |                                     ^^
